### ISSUE TITLE
nvidia-container-cli: detection error: nvml error: unknown error\\\\n\\\"\"": unknown

### ISSUE THREAD

**Issue by hoodiney** [NVIDIA/nvidia-docker#1348]
### 1. Issue or feature description
I just installed nvidia-docker2, and got an error when doing ```sudo docker run --runtime=nvidia --gpus 1 --rm nvidia/cuda:9.0-base nvidia-smi```, outputs are as follows.
```
docker: Error response from daemon: OCI runtime create failed: container_linux.go:349: starting container process caused "process_linux.go:449: container init caused \"process_linux.go:432: running prestart hook 0 caused \\\"error running hook: exit status 1, stdout: , stderr: nvidia-container-cli: detection error: nvml error: unknown error\\\\n\\\"\"": unknown.
```

### 2. Information to [attach](https://help.github.com/articles/file-attachments-on-issues-and-pull-requests/) 
* Distribution version: ubuntu 18.04
* Kernel version: 5.3.0-28-generic 
* Nvidia driver version: 435.21
* Docker version: 19.03.12
* nvidia-container-cli version: 1.2.0

**Comment by klueska** [comment]
That usually points to a driver installation problem. Are you able to run nvidia-smi without docker?

**Comment by hoodiney** [comment]
Thanks for your reply! Yes, I can run nvidia-smi successfully on my host machine. I'm now wondering if this problem is a result of mismatched driver version.

**Comment by hoodiney** [comment]
Problem solved! I retried with driver version 440.1, and it worked!
